include/api.php

```diff
@@ -2463,6 +2464,7 @@ function api_convert_item($item)
 
 	if ($item['title'] != "") {
 		$statushtml = "<br><h4>" . BBCode::convert($item['title']) . "</h4><br>" . $statushtml;
+		$renderedhtml = "<h4>" . BBCode::convert($item['title']) . "</h4>" . $renderedhtml;
```

Comment:
I suggest not adding the title to the body but transmitting the title in a separate field.

Reply:
I would prefer to keep on including the title for backward compatibility reasons. The statusnet_html always had the title included, so I used the same logic for the new friendica_html. In my app code I am using statusnet_html as the fallback if friendica_html is null (on systems with older Friendica versions).

Reply:
There is no backward compatibility thing, since this field is new. Having this as a separate field would allow a client to do special stuff with this field, like having it a link to the whole content or colour it differently.

When the "friendica_html" is available, you can just add the title to it - otherwise you can take the output from "statusnet_html". I really, really would like doing it so.